interface/src/app/node/high_range_exp.tpl.html

```diff
@@ -30,15 +30,43 @@
           <!-- Location of vega-lite widget -->
           <div id="{{ 'exp-' + $index }}"></div>
 
-          <ol ng-if="exp.isExpanded">
-            <li ng-if="!exp.sampleStatus"
-                ng-repeat="sample in exp.samples | orderBy: '-activity'">
-              Activity Value: <b>{{sample.activity.toFixed(5)}}</b><br/>
-              Sample Name: <b>{{sample.name}}</b><br/>
-              Data Source: <b>{{sample.ml_data_source}}</b>
-            </li>
+          <div ng-if="exp.isExpanded">
+            <div ng-if="!exp.sampleStatus">
+              <table class="table table-bordered table-condensed table-hover">
+                <thead>
+                  <tr>
+                    <th>#</th>
+                    <th>Activity</th>
+                    <th>Sample Name</th>
+                    <th>Data Source</th>
+                  </tr>
+                </thead>
+                <tbody>
+                  <tr ng-repeat="sample in exp.samples | orderBy: '-activity'"
+                      ng-class="{success: sample.selected}">
+                    <td>
+                      <label>
+                        <input type="checkbox" ng-model="sample.selected"
+                               ng-change="toggleSelection(exp, sample.selected)">
+                        {{$index + 1}}
+                      </label>
+                    </td>
+                    <td>{{sample.activity.toFixed(5)}}</td>
```

Comment:
I am going to create a new constant service called `activityDigits` (which will be set to 5) in `adage.utils` module, then refer to this constant in both the current directive and `sample_annotation/annotation.js`. I will create an issue for now and file a separate PR later.

We can use the same approach for the P values, if you agree. It wouldn't make sense to set a mandatory constant of precision for all floating point values in the whole app, because different variables may need different precision.